components/chassis.py

```diff
@@ -90,6 +92,17 @@ def get_speed(self):
         return self.drive.getSelectedSensorVelocity() * self.DRIVE_SENSOR_TO_METRES * 10
 
     def set(self, desired_state: SwerveModuleState):
+
+        if abs(desired_state.speed) < 1e-3:
+            if abs(self.last_speed) > 1e-13:
+                self.drive.setIntegralAccumulator(0, 0, 0)
+                self.drive.neutralOutput()
```

Comment:
Might be a good idea to hold a velocity of 0 in case someone tries to push us?

Reply:
I just copied this from the old swerve drive lib. You are correct and I have changed the thing